File: 프로그래머스/0/120809. 배열 두배 만들기/배열 두배 만들기.java
import java.util.stream.Collectors;
import java.util.stream.IntStream;

class Solution {
    public int[] solution(int[] numbers) {
        
        return IntStream.of(numbers)
                        .map(i -> i * 2)
                        .toArray();
    }
}

// 다른 풀이
for(int i=0; i<numbers.length; i++) numbers[i] *= 2;
return numbers;

File: 프로그래머스/0/120809. 배열 두배 만들기/배열 두배 만들기.py
def solution(numbers):
    return list(map(lambda i : i * 2, numbers))

# 참고
# map > list comprehension > map + lambda > for loop 으로 성능이 좋음

# 다른 풀이
# 리스트 컴프리헨션
return [num*2 for num in numbers]

# 다른 풀이
answer = []
for num in numbers:
    answer.append(num * 2)
return answer

